docker-compose-generator/docker-fragments/opt-addnodes.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
```

Comment:
You need to document this in the README.

Reply:
OK